core/src/tile/tileWorker.cpp

```diff
@@ -16,6 +16,8 @@ namespace Tangram {
 TileWorker::TileWorker(int _num_worker) {
     m_running = true;
 
+    m_mainThreadJobQueue.makeCurrentThreadTarget();
+
```

Comment:
Yeah same, I'd rather have exactly one way to set the target thread and not set it implicitly. The default constructor for `std::thread::id` creates an ID that represents no thread, which I think is a sensible initial condition.